projects/core/src/user/store/actions/user-addresses.action.spec.ts

```diff
@@ -1,5 +1,12 @@
-import * as fromUserAddressesAction from './user-addresses.action';
```

Comment:
missing tests for "add/update/delete" actions.

Reply:
Good catch